src/main/java/org/folio/linked/data/integration/kafka/sender/search/AuthorityCreateMessageSender.java

```diff
@@ -35,18 +38,33 @@ public class AuthorityCreateMessageSender implements CreateMessageSender {
 
   @Override
   public Collection<Resource> apply(Resource resource) {
-    return Stream.of(resource)
-      .filter(this::test)
+    if (isAuthority(resource)) {
+      if (isNull(resource.getIndexDate())) {
```

Comment:
@pkjacob should we keep this inNull() check for the case when an authority created through editor?

Reply:
Yes. Let us keep the isNull check.